### DISCUSSION TITLE
Installation Instructions

### DISCUSSION THREAD

**Discussion by Irev-Dev** [Irev-Dev/Round-Anything#21] [General]
Generally speaking the source of truth is the official [OpenSCAD documentation for including libraries](https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Libraries).

But the TL:DR version is that will work for most people is:

1) On this repo's homepage, click "code" -> "Download ZIP"
![image](https://user-images.githubusercontent.com/29681384/106103073-7b806c00-6194-11eb-9236-75a80f6dacfd.png)

2) Unzip the folder and rename to remove "master" from the name, rename to "Round-Anything"
3) Put this folder in the OpenSCAD library folder for your OS, defaults are:
  - Windows: `My Documents\OpenSCAD\libraries`
  - Linux: `$HOME/.local/share/OpenSCAD/libraries`
  - Mac OS X: `$HOME/Documents/OpenSCAD/libraries`
2) Use `<include>` to use the library, the following minimal script to should verify the library is working:
```openscad
include <Round-Anything/polyround.scad>

polygon(polyRound([
  [0, 0, 0],
  [10,0, 1],
  [0, 10,0]
],10));
```

Please note that installing the library this way will mean that you won't receive new updates to the library so checking back once in a while is a good idea, Replace the folder to update the library.

Alternatively, If you're comfortable with git, you can make this repo a submodule of your own repo to make it easy to update.

If you run into trouble feel free to leave a comment on this thread.

**Comment by underwoodblog** [comment]
; is missing in the minimal script example

**Reply by Irev-Dev** [reply]
Thanks @underwoodblog

**Comment by dubbySwords** [comment]
Thank you for the lib. Just not sure how to tell if the minimal script has worked :-(
I am new to openscad.

**Reply by dubbySwords** [reply]
Got it, render f6
